### PR TITLE
Scale - Add ceph_health_check in ceph pod respin test

### PR DESCRIPTION
**Pull request by ramkiperiy**
Updated Scale script to validate ceph status after ceph pod respin
Removed bugzilla markers in pytest params
Updated script to perform memory leak test.

Signed-off-by: Ramakrishnan <rperiyas@redhat.com>

### CODE REVIEW

File: tests/e2e/scale/test_pv_scale_and_respin_ceph_pods.py [hunk 1]
```diff
@@ -183,5 +183,5 @@ def test_pv_scale_out_create_pvcs_and_respin_ceph_pods(
 
         # Added sleep for test case run time and for capturing memory leak after scale
         time.sleep(test_run_time)
-        # TODO: Skipping memory leak analysis because of bz 1750328
-        # helpers.memory_leak_analysis(median_dict)
+        assert utils.ceph_health_check(delay=180), "Failed, Ceph health NOT_OK after ceph pod respin"
```
Comment (RazTamir) [inline tests/e2e/scale/test_pv_scale_and_respin_ceph_pods.py:186]: no need assert as this function returns nothing but raise exception in case of failure

Comment (ramkiperiy) [reply tests/e2e/scale/test_pv_scale_and_respin_ceph_pods.py]: Updated the TC

File: tests/e2e/scale/test_pv_scale_and_respin_ceph_pods.py [hunk 2]
```diff
@@ -183,5 +183,5 @@ def test_pv_scale_out_create_pvcs_and_respin_ceph_pods(
 
         # Added sleep for test case run time and for capturing memory leak after scale
         time.sleep(test_run_time)
-        # TODO: Skipping memory leak analysis because of bz 1750328
-        # helpers.memory_leak_analysis(median_dict)
+        utils.ceph_health_check(delay=180)
```
Comment (ebenahar) [inline tests/e2e/scale/test_pv_scale_and_respin_ceph_pods.py:186]: delay is "Delay in seconds between retries". Is there a reason to have a polling rate of 180 seconds?

Comment (ramkiperiy) [reply tests/e2e/scale/test_pv_scale_and_respin_ceph_pods.py:186]: @ebenahar Setup will have 120 pods with IO and in parallel there will be OSD pod respin, added delay to make sure rebalance is complete and cluster is back to OK state.

Comment (ebenahar) [reply tests/e2e/scale/test_pv_scale_and_respin_ceph_pods.py:186]: @ramkiperiy yes ok but with delay=180, it means that we wait for 3 minutes between checks and this is a very low poll rate. Also, with delay=180 and tries=20 (default value), we are waiting for 180*20 seconds, which is 1 hour

Comment (ramkiperiy) [reply tests/e2e/scale/test_pv_scale_and_respin_ceph_pods.py:186]: ack, This TC doesn't require that much time, will use the default 30sec.

**Pull request merged**